src/App.tsx

```diff
@@ -10,31 +11,67 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, onTabSelected] = useState('tab-1');
```

Comment:
You have an array of tabs. So use it.
```suggestion
  const [selectedTabId, setSelectedTabId] = useState(tabs[0].id);
```